packages/shared/lib/core/ledger/interfaces/ledger-extended-status.interface.ts

```diff
@@ -0,0 +1,6 @@
+import { LedgerStatus } from './ledger-status.interface'
+import { LedgerConnectionState } from './ledger-connection-state.interface'
+
+export interface LedgerExtendedStatus extends LedgerStatus {
```

Comment:
Is there a reason why we're keeping the status and extended status separately?

Reply:
`LedgerStatus` is what wallet-rs returns us and we add the connection status for our purposes.